plugins/com.google.cloud.tools.eclipse.appengine.standard.java8/build.properties

```diff
@@ -1,7 +1,7 @@
-source.. = src/,\
-           src-test/
```

Comment:
The `src-test` folder is pulled in via the `.classpath`.  It's not ideal as it's lost with a _Maven > Update Project…_.  Just writing up a comment about this on on #1847.